inspirehep/modules/theme/jinja2filters.py

```diff
@@ -684,21 +684,19 @@ def format_date(datetext):
 @blueprint.app_template_filter()
 def find_collection_from_url(url):
     """Returns the collection based on the URL."""
-    if 'conferences' in url:
+    if 'cc=conferences' in url or '/conferences' in url:
```

Comment:
I think that a better solution uses something like this: http://stackoverflow.com/a/25013141/374865, rather than parsing the URL.